spec/models/portfolio_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe Portfolio do
+  let(:portfolio)       { FactoryBot.create(:portfolio) }
```

Comment:
There's a really nice pattern where FactoryBot can be included in a spec/support helper.  Then in every test the create and build methods are just globally available, so you just do `create(:portfolio)`.  Thoughts?